modules/showDate.js

```diff
@@ -0,0 +1,18 @@
+/** @format */
+/* eslint-disable */
+import { DateTime } from "./lexon.js";
+
+const showDate = () => {
+	const now = DateTime.now();
+	let content = "";
+	content += `
+<div id="current-date" class="p-2 float-end">
+<h3 class="text-dark">${now}</h3>
+</div>
+`;
```

Comment:
- [ ] The current date is not visible under the navbar, rather is being blocked by the navbar. See image below. Please fix this


![image](https://user-images.githubusercontent.com/84156714/236869683-c7a92461-4294-4b49-be99-1166dd53c57d.png)